updater/lib/dependabot/update_graph_command.rb

```diff
@@ -71,7 +71,7 @@ def build_submission(dependency_snapshot)
         job_id: job.id.to_s,
         branch: job.source.branch || "main",
         sha: dependency_snapshot.base_commit_sha,
-        ecosystem: T.must(dependency_snapshot.ecosystem),
```

Comment:
@brrygrdn was there a reason you went with `dependency_snapshot.ecosystem` here instead of `job.package_manager`? It seems the ecosystem needs something to happen to populate. Might be easier to grab it from the job if there wasn't a reason.

Reply:
Nope, this is better!